app/src/main/java/com/github/sdp/mediato/utility/adapters/CollectionListAdapter.java

```diff
@@ -62,12 +61,10 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
       holder.addMediaButton.setVisibility(View.INVISIBLE);
     }else{
       holder.addMediaButton.setOnClickListener(v -> {
-
-        Review review = s.getMovieReview();
-        collection.addReview(review);
-
         // Can be used to execute some code in the profilePage, such as updating the database
         if (onAddMediaButtonClickListener != null) {
+          Review review = s.getMovieReview();
+          //collection.addReview(review);

```

Comment:
Same remark about commented code, why do we need to keep it ?

Reply:
Removed it